src/setup/rgw.py

```diff
@@ -122,14 +124,31 @@ def config(conf: MappedConf):
             Rgw._update_rgw_config_with_endpoints(conf, service_endpoints, instance)
             instance = instance + 1
 
-        Log.info('Config phase completed.')
-        return 0
+        # Read Motr HA endpoint from data pod using hctl fetch-fids and update in config file
+        Log.info('Reading motr_ha_endpoint from data pod')
+        config_path = Rgw._get_cortx_conf(conf, CONFIG_PATH_KEY)
+        hare_config_dir = os.path.join(config_path, 'hare', 'config', Rgw._machine_id)
+        data_pod_hostname = socket.gethostname().replace('server', 'data')
```

Comment:
put this logic in separate function. We may want to improve this later or reuse even somewhere else

Reply:
done